yarn.lock

```diff
@@ -6096,7 +6087,7 @@ rc-tree@~5.7:
     rc-util "^5.16.1"
     rc-virtual-list "^3.5.1"
 
-rc-util@^5.16.1, rc-util@^5.21.0, rc-util@^5.27.0, rc-util@^5.36.0, rc-util@^5.38.0:
+rc-util@^5.16.1, rc-util@^5.21.0, rc-util@^5.36.0, rc-util@^5.38.0:
```

Comment:
BTW @robyww, I had manually removed `rc-slider` from `package.json` and run `yarn install` to update `yarn.lock`. 

This also ended up removing `rc-util@^5.27.0` and `classnames@^2.2.5` (both versions may be related to `rc-slider`?)...wanted to make sure this is okay.